pkg/validation/spec/info.go

```diff
@@ -89,17 +89,9 @@ func (e Extensions) GetObject(key string, out interface{}) error {
 	return nil
 }
 
-func (e Extensions) sanitize() {
-	for k := range e {
-		if !isExtensionKey(k) {
-			delete(e, k)
-		}
-	}
-}
-
 func (e Extensions) sanitizeWithExtra() (extra map[string]any) {
 	for k, v := range e {
-		if !isExtensionKey(k) {
+		if !internal.IsExtensionKey(k) {
```

Comment:
The reason you keep `sanitizeWithExtra` is that it's not going to be needed in v3?

Reply:
sanitizeWithExtra is only used with spec.Schema's ExtraProps.